graphql_compiler/compiler/context_helpers.py

```diff
@@ -4,6 +4,11 @@
 from ..exceptions import GraphQLCompilationError
 
 
+def is_in_fold_last_level_scope(context):
```

Comment:
how about `innermost_scope` instead of `last_level_scope` everywhere? I think it's more precise since `level` is never formally defined anywhere